### PR TITLE
Lose extra whitespace

### PR DESCRIPTION
**Pull request by inducer**

### CODE REVIEW

**Comment by inducer** [comment]
@conda-forge/core Something seems wrong with the config of the windows autobuild here--this issue seems like it's happening _way_ before anything recipe-specific comes in:

https://ci.appveyor.com/project/conda-forge/pyvisfile-feedstock/build/1.0.4/job/4698tdyu5ea2itku

How do I fix that?

**Comment by inducer** [comment]
Forgot to say: Thanks in advance for the help!

**Comment by patricksnape** [comment]
This is a known bug in conda at the moment I think - bare with us.

**Comment by jakirkham** [comment]
Please [re-render](https://github.com/conda-forge/conda-smithy#re-rendering-an-existing-feedstock) with `conda-smithy` version `1.3.3`. This should fix that issue.

**Comment by inducer** [comment]
Done, thx!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ source:
     sha256: {{ sha256 }}
 
 build:
-    number: 0
+    number: 1
```
Comment (jakirkham) [inline recipe/meta.yaml]: No need for the build number bump.

Comment (inducer) [reply recipe/meta.yaml]: Despite the rerender?

Comment (jakirkham) [reply recipe/meta.yaml]: Yeah, re-rendering doesn't normally get a build number bump. We have had to do build number bumps sometimes with re-renders if there are changes to the recipe or if we actually need a new package deployed.

**Comment by jakirkham** [comment]
Thanks @inducer .

**Pull request merged**